dev/one-page.md

```
@@ -1,15 +1,18 @@
 ---
 
 <script setup lang="ts">
 import { useRoute, useData } from 'vitepress'
+import ThemeConfiguration from './.vitepress/theme/components/ThemeConfiguration.vue'
 
 const route = useRoute()
 
 const { isDark } = useData()
 </script>
 
+<ThemeConfiguration />
```

Markdownlint

Element: ThemeConfiguration
Inline HTML

(MD033, no-inline-html)